window/src/os/windows/window.rs

```diff
@@ -815,6 +815,104 @@ unsafe fn wm_ncdestroy(
     None
 }
 
+unsafe fn wm_nccalcsize(
+    hwnd: HWND,
+    _msg: UINT,
+    _wparam: WPARAM,
```

Comment:
The underscore prefix is used to indicate unused parameters, but these parameters are actually used!

```suggestion
    wparam: WPARAM,
```